observability/opentelemetry/src/main/java/org/wildfly/extension/opentelemetry/OpenTelemetrySubsystemAdd.java

```diff
@@ -34,25 +38,41 @@ class OpenTelemetrySubsystemAdd extends AbstractBoottimeAddStepHandler {
     protected void performBoottime(OperationContext context, ModelNode operation, ModelNode model) throws OperationFailedException {
         super.performBoottime(context, operation, model);
 
+        String exporter = OpenTelemetrySubsystemDefinition.EXPORTER.resolveModelAttribute(context, model).asString();
+
+        if (EXPORTER_JAEGER.equals(exporter)) {
+            if (context.isNormalServer()) {
+                throw new OperationFailedException(OTEL_LOGGER.jaegerIsNoLongerSupported());
+            } else {
+                OTEL_LOGGER.warn(OTEL_LOGGER.jaegerIsNoLongerSupported());
+            }
+        }
```

Comment:
Please convert this to a static method so it can be used for write-attribute as well. And then OpenTelemetrySubsystemDefinition would need to override registerAttributes with a new implementation that handles EXPORTER differently with a handler that calls the static function.

BTW, this comment is what led me to start this thread: https://wildfly.zulipchat.com/#narrow/stream/174184-wildfly-developers/topic/Write.20attribute.20handler.20registration

Reply:
To be more clear, I'm referring to extracting https://github.com/wildfly/wildfly/pull/17397/files#diff-72d387b5d42a4c7352e43b0c764284eb154c6e34e15d6e9009d6d4867089a943R43-R49 out into a method.

Reply:
Done.